locale/de-de/last.voc

```diff
@@ -0,0 +1,33 @@
+letzt
+letzte
+letzten
+letztes
+letzter
+letztem
+letztere
+letzterer
+letzteres
+hinter
+hintere
+hinteren
+hinteres
+hinterer
+hinterem
+abschließend
+abschließende
+abschließenden
+abschließendes
+abschließender
+abschließendem
+schließend
+schließende
+schließenden
+schließendes
+schließender
+schließendem
+ende
+enden
+endes
+ender
```

Comment:
would 5 slots suffice here? using the optional syntax i would group these as

(letzt|...)
(hinter|....)
(abschließend|...)
(schließend|...)
(ende|...)

?

Reply:
yes that works, i tested it. btw it also works without the parentheses

Reply:
It also works in just one line, so you maybe don't have to add more lines to the english file

Reply:
I will try to put them in one line wherever possible, will tell you when it is not